docs/auth-modes.md

```diff
@@ -228,6 +228,42 @@ Note: the ```value``` attribute value for all options is customizable by connect
 })
 ```
 
+### Database impersonation using embedded credentials (DelegationUID)
+
+This applies to databases which support a user that can delegate requests from other user(s). Databases support this by enabling the client to pass a DelegationUID to the server.
+In the case of Tableau server, you can pass the identity of the logged-in user on Tableau Server as the DelegationUID. Tableau would then pass that property to the database using driver properties and authentication is taken care of by the database. This community [article](https://community.tableau.com/docs/DOC-11137) gives more information on Database Impersonation using Embedded credentials.   
```

Comment:
Does it work/make sense on desktop? Do you have to do extra work in the connector files to use it on server?

Reply:
Delegation is useful on Tableau Server. We do not support it on Desktop. On Desktop the user would just login using their username-password. 
The changes in connectionProperties and manifest are required to be present at the server level, however desktop plugin can be just a simple jdbc plugin supporting Basic Auth

Reply:
May want to re-word the description to say that explicitly, the phrase "in the case of Tableau Server" implies there is a Desktop case.